packages/@ember/-internals/glimmer/tests/integration/application/engine-test.js

```diff
@@ -81,7 +81,9 @@ moduleFor(
         'engine:blog',
         Engine.extend({
           init() {
-            this._super(...arguments);
+            ignoreDeprecation(() => {
```

Comment:
Hmm, its not clear why we hit the assertion here. AFAICT, this test isn't about using the deprecated resolver, so it should probably be updated to avoid the deprecation (vs ignoring it).

Reply:
So basically, if we extend Engine without supplying a Resolver other than DefaultResolver, we hit the deprecation.